packages/react-centra-checkout/src/Context/index.tsx

```diff
@@ -60,6 +60,7 @@ export interface ContextMethods {
     id: string,
     newPassword: string,
   ): Promise<Centra.SelectionResponseExtended>
+  resetSelection?(): void
```

Comment:
Looks good! Let's just add a documentation comment to explain what this method does and why it's needed.